lib/client/identityfile/identity.go

```diff
@@ -614,10 +614,15 @@ func KeyFromIdentityFile(identityPath, proxyHost, clusterName string) (*client.K
 // is also added to its profile store with the limited profile data available
 // in the identity file.
 //
-// Since identity files do not save a proxy address, proxyAddr must be provided
-// to fill in this data gap. clusterName can also be provided to aim the key at
-// a leaf cluster rather than the default root cluster.
+// proxyAddr: Since identity files do not save a proxy address, a host:port-like
```

Comment:
This `arg: description` format is not super common. I would just refer to the argument in plain sentences.

Maybe something like this:

```
// Use [proxyAddr] to specify the address of the proxy.
// This is necessary because identity files do not store
// the proxy address. Additionally, the optional [clusterName]
// argument can be used to target a leaf cluster rather than
// the default root cluster.
```

Reply:
Seems super reasonable. I'll ammend with a format like you suggest.